### DISCUSSION TITLE
Conflict with plugin PremiumHub

### DISCUSSION THREAD

**Discussion by FZDNSZYT** [Zrips/CMI#7408] [General]
How to disable the death point map, the lobby fixed menu of this plugin PremiumHub cannot have other things by default, but after the player is killed, a death map will be forced into it. This causes errors. I wish to disable this feature in the login lobby.

**Comment by CloudeLecaw** [comment]
cmi doesnt give you a death map. so it must be an other plugin.

The only two options you got from cmi is pointing the deathcords in chat on death in a meassage what can be prevented by not giving the permission.
and to set a symbol of your deathpoint to the cmi bossbar compass, what can be disabled in config if not wanted. 

there exists three other options to get players deathcords too...cmi invcheck, cmi info check and inv invlist if you let cmi save your inventory on death. but these all are more administrative features that players normaly should not have.

well at least none of these features give you a deathpointmap on death. so take a look at your other plugins please what gives you a deathpointmap.

waiting for your response :-)

**Reply by FZDNSZYT** [reply]
Sorry, because CMI is fully functional, I thought it was CMI, and I have tested that it is the function of itemsadder

**Reply by FZDNSZYT** [reply]
Another problem is that after the lobby's void hub is protected by a Residence circle, other players cannot log in normally. After they join the game, they will keep falling freely, directly passing through the ground, and cannot land. How to describe it, you should have seen the movie The Avengers 4: Endgame. After they joined the game, they were as miserable as Loki being played by Doctor Strange. Hahaha

**Reply by FZDNSZYT** [reply]
After I switched to WorldGuard protection, the problem was fixed

**Reply by CloudeLecaw** [reply]
> Another problem is that after the lobby's void hub is protected by a Residence circle, other players cannot log in normally. After they join the game, they will keep falling freely, directly passing through the ground, and cannot land. How to describe it, you should have seen the movie The Avengers 4: Endgame. After they joined the game, they were as miserable as Loki being played by Doctor Strange. Hahaha

this is a ressidence thing not a cmi thing :)

> After I switched to WorldGuard protection, the problem was fixed

also ask please in the ressidence github for a solution here:)

**Reply by FZDNSZYT** [reply]
okey
